resources/views/structure/footer.blade.php

```diff
@@ -0,0 +1,9 @@
+<footer>
+    <div class="container">
+        <p class="float-right">
+            <a href="#">Back to top</a>
+        </p>
+        <p>Album example is &copy; Bootstrap, but please download and customize it for yourself!</p>
```

Comment:
I believe this footer note doesn't belong here...

Reply:
Yes, indeed.